src/tasks/tasks.service.ts

```diff
@@ -6,22 +6,42 @@ import { TasksRepository } from './tasks.repository';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Task } from './task.entity';
 import { User } from '../auth/user.entity';
+import { PrismaService } from '../prisma.service';
+import { TasksPrismaService } from '../tasks-prisma.service';
+import { task } from '@prisma/client';
+import { user } from '@prisma/client';
```

Comment:
I think we could use a single import for these variables 🤔
```suggestion
import { task, user } from '@prisma/client';
```